src/lib/components/Layout/Col.svelte

```diff
@@ -0,0 +1,38 @@
+﻿<div {...props}>
+  <slot />
+</div>
+
+<script lang="ts">
+  type colSpan = number | string | undefined;
+
+  export let col: colSpan = undefined;
+  export let sm: colSpan = undefined;
+  export let md: colSpan = undefined;
+  export let lg: colSpan = undefined;
+
```

Comment:
No blank line in props declartion, tho I understand the reasoning behind, grouping is enough for me
```suggestion
```